src/plots/gl2d/camera.js

```diff
@@ -262,34 +262,27 @@ function createCamera(scene) {
         var lastX = result.lastPos[0],
             lastY = result.lastPos[1];
 
-        switch(scene.fullLayout.dragmode) {
```

Comment:
Why is this gone? `fullLayout.dragmode: false` should still be a thing.

Reply:
From what I can tell, `dragmode: false` isn't an option, is it? If I set that, it defaults to `zoom`. This is removed because the dragmode should not be affecting mousewheel behavior at all.

Reply:
Ahhh possible bug…… 🕷 

When setting `dragmode: false` with gl3d, dragging is disabled but `gd._fullLayout.dragmode = 'zoom'`. That suggests it's sanitizing it but using `gd.layout.dragmode` instead of `gd._fullLayout.dragmode`.

Reply:
Looks like we only support `layout.scene.dragmode: false` (ie 3D), not `layout.dragmode: false`
@rreusser good catch that this should not be disabled in zoom mode anyway.

Reply:
The problem then is that `dragmode: false` is not a valid coercible default. `gl3d` seems to sidestep the coerced default and look at the user data instead of sanitized data. Which works, though it's perhaps a bit sketchy.

Reply:
Oh, I see. It *inherits* false from `layout.dragmode` despite `layout.dragmode: false` not actually being a valid default. I'm okay with that.

Reply:
Oopppppppps, that line is in `gl2d/camera.js (i.e. **NOT** gl3d). Funny that thing has been there since [v1.0.0](https://github.com/plotly/plotly.js/blame/master/src/plots/gl2d/camera.js#L110). 

:hocho: :hocho: :hocho:  :hocho:  :hocho:  :hocho:

Reply:
Yeah. Definitely undesirable. 😄 🔪

Reply:
(next most glaring thing, as briefly mentioned to @alexcjohnson is double-click to revert zoom to autoscale, which is the top top top thing that always gets me about scattergl plots.)

Reply:
> which is the top top top thing that always gets me about scattergl plots.)

No need to waste time on this. Double-click will _just work_ after @dfcreative 's https://github.com/plotly/plotly.js/pull/1869

Reply:
Works for me. Should have bothered to do it long ago, but I'm content to wait for #1869. 🎉